repos/system_upgrade/el7toel8/actors/satelliteupgradecheck/actor.py

```diff
@@ -0,0 +1,22 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.satelliteupgradecheck import satelliteupgradecheck
+from leapp.models import Report, SatelliteFacts
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class SatelliteUpgradeCheck(Actor):
+    """
+    Check state of Satellite system before upgrade
+    """
+
+    name = 'satelliteupgradecheck'
```

Comment:
Could you please change it to 'satellite_upgrade_check' for better readability?

Reply:
This is autogenerated